app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt

```diff
@@ -43,9 +43,19 @@ class PatientSummaryEffectHandler @AssistedInject constructor(
             uiWorkScheduler = schedulersProvider.ui()
         ))
         .addTransformer(LoadCurrentFacility::class.java, loadCurrentFacility(schedulersProvider.io()))
+        .addTransformer(HandleEditClick::class.java, handleEditClick(uiWorkScheduler = schedulersProvider.ui()))
```

Comment:
This can be replaced with a `consumer` instead.

```kotlin
.addConsumer(HandleEditClick::class.java, { uiActions.showPatientEditScreen(it.patientSummaryProfile) }, schedulersProvider.ui())
```